tox.ini

```diff
@@ -50,31 +55,31 @@ passenv =
     CI
     GITHUB*
 
-[qadocs]
-basepython = python3
-envdir = {toxworkdir}/python3
-deps =
-    flake8
-    sphinx
-
```

Comment:
No longer using common "qa" and "docs" testenv because "docs" now invokes pytest (for doctesting), and having pytest as "qa" dep is much too heavy...